client/user/src/app/(nonRoot)/profile/order/_components/OrderCard.tsx

```
@@ -0,0 +1,186 @@
+              size="md"
+              className="mt-4 w-full"
+              onClick={() => closeModal()}
+            >
+              확인
+            </Button>
+          </div>
+        </Confirm>
+      )
+    } else if (status === 'COMPANY_ARRIVED') {
+      openModal(<></>)
+    } else if (status === 'PAYMENT_COMPLETED') {
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)